packages/core/src/components/forms/_label.scss

```diff
@@ -136,21 +139,21 @@ label.pt-label {
 
   &.pt-disabled {
     &,
-    .pt-label-description {
+    .pt-muted {
       color: $pt-text-color-disabled;
     }
   }
 
   .pt-dark & {
     color: $pt-dark-heading-color;
 
-    .pt-label-description {
+    .pt-muted {
       color: $pt-text-color-muted;
     }
 
     &.pt-disabled {
       &,
-      .pt-label-description {
+      .pt-muted {
```

Comment:
wait what's `pt-muted`? this is also a new modifier.

**did you mean `pt-text-muted`?**

if you did mean `pt-text-muted` then you don't need to set the original color, only the disabled color.